experimental/listener.go

```diff
@@ -6,6 +6,24 @@ import (
 	"github.com/tetratelabs/wazero/api"
 )
 
+// StackIterator allows iterating on each function of the call stack, starting
+// from the top. At least one call to Next() is required to start the iteration.
+//
+// Example:
```

Comment:
maybe add another example in the example test ( or make stackiterator_example_test.go)  that does something like this, then remove this comment.

Reply:
Done